pages/cloudflare/migrate-from-0.3.mdx

```diff
@@ -0,0 +1,38 @@
+import { SITE } from '../../config';
+import { Callout } from 'nextra/components';
+
+### Migrate from 0.3
+
+The `@opennextjs/cloudflare@0.4.0` introduced a new `initOpenNextCloudflareForDev` utility and made `getCloudflareContext` synchronous,
+we'll explore those two differences below, and how they effect applications built using `0.3.x` versions of the adapter.
+
+##### `initOpenNextCloudflareForDev`
+
+`initOpenNextCloudflareForDev` is a new utility that needs to be added to the Next.js configuration file in order to integrate the adapter
+with the Next.js dev server. If you don't plan on using the `next dev` command you can skip this section, otherwise update your Next.js
+configuration file to import and call the utility.
+
+Example:
+```js
+// next.config.mjs
+
+import { initOpenNextCloudflareForDev } from "@opennextjs/cloudflare";
+
+initOpenNextCloudflareForDev();
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {};
+
+export default nextConfig;
+```
+
+##### Synchronous `getCloudflareContext` synchronous
+
+`getCloudflareContext` has been changed not to synchronously return the Cloudflare context object instead or returning
```

Comment:
There was an extra **not** - I also propose a shorter version. Apply the fix that you prefer :)

```suggestion
`getCloudflareContext` is now synchronous.
```

Reply:
yeah and there's also a typo there 😓 :+1: